apps/table.html

```diff
@@ -372,7 +372,9 @@ <h5 class="modal-title" id="deleteModalLabel">Delete confirmation</h5>
 							else if (!d[key]) rs.push('')
 							else rs.push(d[key])
 						});
-						const btn = `<div id='open-delete'> <button class="btn btn-success" data-id='${rs[0]}' onclick='openView(this)'>Open</button> <button type='button' class='btn btn-danger DelButton' id='deleteBtn' data-id='${rs[0]}' data-name='${rs[1]}' onclick='deleteSld(this)' data-toggle='modal'> <i class='fas fa-trash-alt' ></i> </button></div>`
+						const btn = `<div id='open-delete'> <button class="btn btn-success" data-id='${rs[0]}' onclick='openView(this)'>Open</button> 
+							<button type='button' class='btn btn-danger DelButton' id='deleteBtn' data-id='${rs[0]}' data-name='${rs[1]}' onclick='deleteSld(this)' data-toggle='modal'> <i class='fas fa-trash-alt' ></i> </button>
+							<button type='button' class='btn btn-info DownloadButton' id='downloadBtn' data-id='${rs[0]}' onclick='handleDownload(this)' > <i class='fas fa-download' ></i> </button></div>`
```

Comment:
```suggestion
							<button type='button' class='btn btn-info DownloadButton' id='downloadBtn' data-id='${rs[0]}' onclick='downloadSlide(this)' > <i class='fas fa-download' ></i> </button>
							<button type='button' class='btn btn-danger DelButton' id='deleteBtn' data-id='${rs[0]}' data-name='${rs[1]}' onclick='deleteSld(this)' data-toggle='modal' style='display:none'> <i class='fas fa-trash-alt' ></i> </button> </div>`
```
These 2 changes can be made 
- The delete icon should not be displayed initially, as we are calling checkUserPermissions() in line 379 of table.html , which will automatically change the 'display' to 'block', iff the user has apt. permissions.
- The order of the download icon and delete icon can be reversed, so that the delete icons get shifted to the rightmost